src/configurations/apiSchema.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * Schema on Notion:
+ *   https://www.notion.so/primedao/PrimeDeals-Data-Module-Scheme-378f2f66edfe423f9660ba32a26e62ad
+ */
+
+export interface IPrimeDeals {
```

Comment:
looks like some of these interfaces (or parts of it) where created in `src/dealWizard/dealWizard.types.ts` and might be duplicated. Is it the case?

Reply:
> duplicated

Yeah possible, but I wanted to have the diff for this PR minimal.
So removing the duplicates would be another PR.

Reply:
got it. Not to forget